velox/connectors/hive/iceberg/CMakeLists.txt

```diff
@@ -19,6 +19,6 @@ add_library(
 target_link_libraries(velox_hive_iceberg_splitreader velox_connector
                       Folly::folly)
 
-if(${VELOX_BUILD_TESTING})
+if(${VELOX_BUILD_TESTING} AND VELOX_ENABLE_PARQUET)
```

Comment:
Why Iceberg need to be coupled with Parquet?  You only need Parquet for testing

Reply:
@Yuhta IcebergSplitReader benchmark & benchmark tests are reading and writing Parquet data so it depends on VELOX_ENABLE_PARQUET. Let me know if in this case, I should shift the benchmark & its tests to different directory and then use `VELOX_ENABLE_PARQUET`?